### ISSUE TITLE
No topics for text scrapped from websites.

### ISSUE THREAD

**Issue by mirfan899** [MaartenGr/BERTopic#671]
I am trying to label topics of a website pages. No topics extracted on around 100 pages of texts. I tried to use different models but the results are same. Attaching the text file for reference.
[texts.txt](https://github.com/MaartenGr/BERTopic/files/9315494/texts.txt)

```python
from bertopic import BERTopic
import os

os.environ["TOKENIZERS_PARALLELISM"] = "false"

docs = open("texts.txt").readlines()

# topic_model = BERTopic(embedding_model="distiluse-base-multilingual-cased-v1", low_memory=True)
topic_model = BERTopic(language="multilingual", low_memory=True)
topics, probs = topic_model.fit_transform(docs)
print(topic_model.get_topic_info())
```
and the output
```shell
/home/irfan/.pyenv/versions/TES/bin/python /home/irfan/PycharmProjects/tes-search-2/bert_test.py 
   Topic  Count               Name
0     -1    100  -1_seo_the_to_and

Process finished with exit code 0
```

What is wrong here?

**Comment by MaartenGr** [comment]
It is likely due to the size of your dataset. 100 documents are not much and require some tuning in HDBSCAN. So setting the `min_topic_size` a bit lower would help. Other than that, using k-Means instead typically works a bit better in small datasets.

**Comment by MaxGSEO** [comment]
> It is likely due to the size of your dataset. 100 documents are not much and require some tuning in HDBSCAN. So setting the `min_topic_size` a bit lower would help. Other than that, using k-Means typically works better in small datasets.

@MaartenGr Any other idea to do topic modeling after crawling a website to scrape its pages? I thought BERTopic was the best solution. Could just collecting article titles (say 500) and using SBERT be a solution? Thanks

**Comment by MaartenGr** [comment]
@MaxGSEO Generally, choosing clustering models that work well with small datasets definitely helps, like using k-Means instead of HDBSCAN since HDBSCAN can generate many outliers. SBERT is the default package in BERTopic, so that should be no problem.
